MadameMaths/CheckUserAnswer.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using System;
 namespace MadameMaths
 {
-    public static class GiveSolution
+    public static class CheckUserAnswer
     {
-        public static int WrongOrRight(double correctAnswer, double danswer, int score)
+        public static int CheckingUserAnswer(double correctAnswer, double finalAnswer, int score)
```

Comment:
@SabBabQab You already have another method called CheckUserAnswer. This one can be removed, no?